clickhouse_driver/columns/ipcolumn.py

```diff
@@ -0,0 +1,64 @@
+from ipaddress import IPv4Address, IPv6Address, AddressValueError
+
+from .. import errors
+from ..util import compat
+from .exceptions import ColumnTypeMismatchException
+from .stringcolumn import ByteFixedString
+from .intcolumn import UInt32Column
+
+
+class IPv4(UInt32Column):
+    ch_type = "IPv4"
+    py_types = compat.string_types + (IPv4Address, int)
+
+    def check_item_type(self, value):
```

Comment:
It seems than `clickhouse_driver.columns.base.Column` has the same `check_item_type`.

Reply:
Yes, I have removed it. However IntColumn and UIntColumn override the types_check methods, so I've added a condition in the __init__ to properly handle types_check=True on IP columns (added matching tests)